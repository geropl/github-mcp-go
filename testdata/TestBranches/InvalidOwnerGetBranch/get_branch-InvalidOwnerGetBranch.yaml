---
version: 2
interactions:
    - id: 0
      request:
        proto: HTTP/1.1
        proto_major: 1
        proto_minor: 1
        content_length: 0
        transfer_encoding: []
        trailer: {}
        host: api.github.com
        remote_addr: ""
        request_uri: ""
        body: ""
        form: {}
        headers:
            Accept:
                - application/vnd.github.v3+json
            User-Agent:
                - go-github/v69.2.0
            X-Github-Api-Version:
                - "2022-11-28"
        url: https://api.github.com/repos/invalid-owner/github-mcp-go-test/branches/main
        method: GET
      response:
        proto: HTTP/2.0
        proto_major: 2
        proto_minor: 0
        transfer_encoding: []
        trailer: {}
        content_length: -1
        uncompressed: true
        body: '{"message":"Not Found","documentation_url":"https://docs.github.com/rest/branches/branches#get-a-branch","status":"404"}'
        headers:
            Access-Control-Allow-Origin:
                - '*'
            Access-Control-Expose-Headers:
                - ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset
            Content-Security-Policy:
                - default-src 'none'
            Content-Type:
                - application/json; charset=utf-8
            Date:
                - Fri, 07 Mar 2025 15:28:15 GMT
            Github-Authentication-Token-Expiration:
                - 2025-04-06 13:14:03 +0200
            Referrer-Policy:
                - origin-when-cross-origin, strict-origin-when-cross-origin
            Server:
                - github.com
            Strict-Transport-Security:
                - max-age=31536000; includeSubdomains; preload
            Vary:
                - Accept-Encoding, Accept, X-Requested-With
            X-Accepted-Github-Permissions:
                - contents=read
            X-Content-Type-Options:
                - nosniff
            X-Frame-Options:
                - deny
            X-Github-Api-Version-Selected:
                - "2022-11-28"
            X-Github-Media-Type:
                - github.v3; format=json
            X-Github-Request-Id:
                - 2258:3C7982:43E2A88:46039A9:67CB108F
            X-Ratelimit-Limit:
                - "5000"
            X-Ratelimit-Remaining:
                - "4958"
            X-Ratelimit-Reset:
                - "1741364247"
            X-Ratelimit-Resource:
                - core
            X-Ratelimit-Used:
                - "42"
            X-Xss-Protection:
                - "0"
        status: 404 Not Found
        code: 404
        duration: 128.158666ms
