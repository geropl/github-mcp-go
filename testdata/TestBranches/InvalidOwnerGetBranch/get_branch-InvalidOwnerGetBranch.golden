{
  "output": "",
  "err": "GitHub API Error: GitHub API error: unexpected status code: 404 Not Found"
}